data/json/items/gun/223.json

```diff
@@ -97,7 +97,11 @@
     "sight_dispersion": 60,
     "recoil": 75,
     "durability": 8,
-    "burst": 15,
+    "modes": [
+      [ "DEFAULT", "semi-auto", 1 ],
+      [ "BURST", "2 rd.", 2 ],
```

Comment:
Although, it may be add in the future. Would anybody support a rename to G36A1 or something?